src/components/AdminDashboard/CompleteIncident.js

```diff
@@ -69,6 +70,8 @@ const CompleteIncident = props => {
   return (
     <div className="complete-incident">
       <div className="complete-incident-dropdown">
+
+        {/* Date */}
         {!editing ? (
```

Comment:
Later could we perhaps move the form and display elements into respective components and write this similar to (maybe with better names): 
```jsx
{!editing ? <Form/> : <DisplayElems/>}
```
instead of repeating that ternary statement?

Reply:
Yeah, I agree. I'll split that up right now.

Reply:
I didn't split it up into separate components as this PR is already sprawling... but I did combine the ternary statements

Reply:
Yea, I think that could be put in another PR/ticket/whatever. We can get it later.